### ISSUE TITLE
Exception: NoMethodError: undefined method `logger' for ActiveModelSerializers::SerializableResource:Class

### ISSUE THREAD

**Issue by turboladen** [rocketjob/rails_semantic_logger#25]
I'm running Rails 5.0.0.1 with active_model_serializers 0.10.2. I'm trying to get UDP logging going so I can send logs to my logstash server (logstash-logger removes data I'm adding to my log payload, so I'm trying to find another way to use logstash), but I'm getting the described exception.

I've got the following in my `config/environments/development.rb`:

```ruby
# All of these result in the same failure
# SemanticLogger.add_appender appender: :udp, server: 'my_address_and_port'
# SemanticLogger.add_appender logger: config.logger
config.after_initialize do
  config.semantic_logger.add_appender appender: :udp, server: 'my_address_and_port'
end

# The following doesn't seem to change anything, but including just in case
config.rails_semantic_logger.semantic = false
```

Here's the top of the trace in case it's helpful:

```
2016-10-10 16:15:16.670190 F [73913:70130055111100 debug_exceptions.rb:7] [bfae6a2c-464a-4f09-a02d-95583a43a266] Rails -- Exception: NoMethodError: undefined method `logger' for ActiveModelSerializers::SerializableResource:Class
/Users/sloveless/Development/agrian/choopa/vendor/bundle/ruby/2.2.0/gems/semantic_logger-3.3.0/lib/semantic_logger/loggable.rb:44:in `logger'
/Users/sloveless/Development/agrian/choopa/vendor/bundle/ruby/2.2.0/gems/rails_semantic_logger-3.3.1/lib/rails_semantic_logger/extensions/active_model_serializers/logging.rb:9:in `tag_logger'
/Users/sloveless/Development/agrian/choopa/vendor/bundle/ruby/2.2.0/gems/active_model_serializers-0.10.2/lib/active_model_serializers/logging.rb:20:in `block in instrument_rendering'
/Users/sloveless/Development/agrian/choopa/vendor/bundle/ruby/2.2.0/gems/activesupport-5.0.0.1/lib/active_support/callbacks.rb:391:in `instance_exec'
/Users/sloveless/Development/agrian/choopa/vendor/bundle/ruby/2.2.0/gems/activesupport-5.0.0.1/lib/active_support/callbacks.rb:391:in `block in make_lambda'
/Users/sloveless/Development/agrian/choopa/vendor/bundle/ruby/2.2.0/gems/activesupport-5.0.0.1/lib/active_support/callbacks.rb:285:in `call'
/Users/sloveless/Development/agrian/choopa/vendor/bundle/ruby/2.2.0/gems/activesupport-5.0.0.1/lib/active_support/callbacks.rb:285:in `block in halting'
/Users/sloveless/Development/agrian/choopa/vendor/bundle/ruby/2.2.0/gems/activesupport-5.0.0.1/lib/active_support/callbacks.rb:447:in `call'
/Users/sloveless/Development/agrian/choopa/vendor/bundle/ruby/2.2.0/gems/activesupport-5.0.0.1/lib/active_support/callbacks.rb:447:in `block in around'
/Users/sloveless/Development/agrian/choopa/vendor/bundle/ruby/2.2.0/gems/activesupport-5.0.0.1/lib/active_support/callbacks.rb:455:in `call'
/Users/sloveless/Development/agrian/choopa/vendor/bundle/ruby/2.2.0/gems/activesupport-5.0.0.1/lib/active_support/callbacks.rb:455:in `call'
/Users/sloveless/Development/agrian/choopa/vendor/bundle/ruby/2.2.0/gems/activesupport-5.0.0.1/lib/active_support/callbacks.rb:101:in `__run_callbacks__'
/Users/sloveless/Development/agrian/choopa/vendor/bundle/ruby/2.2.0/gems/activesupport-5.0.0.1/lib/active_support/callbacks.rb:750:in `_run_render_callbacks'
/Users/sloveless/Development/agrian/choopa/vendor/bundle/ruby/2.2.0/gems/activesupport-5.0.0.1/lib/active_support/callbacks.rb:90:in `run_callbacks'
/Users/sloveless/Development/agrian/choopa/vendor/bundle/ruby/2.2.0/gems/active_model_serializers-0.10.2/lib/active_model_serializers/logging.rb:68:in `block (2 levels) in notify'
/
```

I suppose there could be something with my configuration

**Comment by turboladen** [comment]
In fact, if I remove any configuration stuff with regard to rails_semantic_logger, the error happens as long as I leave the `gem` entry in my `Gemfile`.

**Comment by kevintyll** [comment]
I believe this is already fixed in master.

**Comment by turboladen** [comment]
@kevintyll confirmed! Thank you!

**Comment by reidmorrison** [comment]
Published ruby gem V3.4.0 with this fix from master
